include/programs.h

```diff
@@ -33,8 +34,6 @@
 #define WIKI_URL                   "https://github.com/dosbox-staging/dosbox-staging/wiki"
 #define WIKI_ADD_UTILITIES_ARTICLE WIKI_URL "/Add-Utilities"
 
-constexpr int autoexec_maxsize = 4096;
-
```

Comment:
Huh, good riddance 😅 I'm hoping we're doing dynamic buffer sizings now!

Reply:
Yes, I have switched to using `std::vector` and `std::string`.